okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java

```diff
@@ -38,6 +38,7 @@
   private String mapName = "modules";
   private LockedTypedMap1<ModuleDescriptor> modules
     = new LockedTypedMap1<>(ModuleDescriptor.class);
+  private Map<String,ModuleDescriptor> enabledModules = new HashMap<>();
```

Comment:
Is this a cache? If yes, can you rename it to `enabledModulesCache`?

Reply:
yep